### PR TITLE
Update P2P dependencies for the new UI and config the practitioner register and profile (PR not atomic)

### PR DESCRIPTION
**Pull request by dubdabasoduba**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #[issue number] 

**Checklist**
- [ ] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [ ] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [ ] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [ ] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/quest/src/main/assets/configs/app/profiles/other_registers_profile_config.json [hunk 1]
```diff
@@ -391,11 +412,54 @@
...
     {
       "title": "Enroll to FP",
+<<<<<<< HEAD
+      "titleColor": "#000000",
+      "visible": "@{isEligibleForFP}",
+=======
       "titleColor": "@{patientTextColor}",
       "visible": "@{canEnrollToFp}",
       "enabled": "@{patientActive}",
+>>>>>>> 54e8ca1fef84b86a4a0722f7f874a93ec75d7c7d
```
Comment (ellykits) [inline android/quest/src/main/assets/configs/app/profiles/other_registers_profile_config.json:462]: Fix merge conflicts.

File: android/quest/src/main/assets/configs/app/application_config.json [hunk 2]
```diff
@@ -2,14 +2,16 @@
...
   "remoteSyncPageSize": 100,
   "languages": [
     "en"
   ],
   "useDarkTheme": false,
   "syncInterval": 30,
-  "syncStrategies": ["Organization", "Location", "CareTeam", "Practitioner"],
+  "syncStrategy": [
+    "Location"
+  ],
```
Comment (ellykits) [inline android/quest/src/main/assets/configs/app/application_config.json:14]: Syncing by Location syncs all data from staging server. Sync has not been optimized yet so this takes over 50mins to complete; not ideal for development.

**Review by ellykits — COMMENTED** [review]
Should we create new configurations for the supervisor app? As long as the app Id and name of directory is different it is okay we can still load them from assets. Or is it fine to mix the configs and use same app debug id?

File: android/quest/src/main/assets/configs/app/application_config.json (hunk 2 above)
Comment (dubdabasoduba) [reply android/quest/src/main/assets/configs/app/application_config.json:14]: > Should we create new configurations for the supervisor app? As long as the app Id and name of the directory are different it is okay we can still load them from assets. Or is it fine to mix the configs and use the same app debug id?

- My thinking has been we use the config directory to hold different variants of the modules we have. I am okay with all being in the same place. We have however separated them on the fhir-resource repo

Comment (dubdabasoduba) [reply android/quest/src/main/assets/configs/app/application_config.json:14]: > Syncing by Location syncs all data from the staging server. Sync has not been optimized yet so this takes over 50mins to complete; not ideal for development.

Makes sense. Let update it

**Review by ellykits — APPROVED** [review]

**Pull request merged**